webapp/src/js/controllers/messages-content.js

```diff
@@ -205,7 +186,7 @@ angular.module('inboxControllers').controller('MessagesContentCtrl',
               ctrl.allLoaded = conversation.length < 50;
               delete ctrl.firstUnread;
             }
-            markAllRead();
+            ctrl.markConversationRead();
```

Comment:
is this functionally equivalent? replacing `markAllRead()` with a call to `markSelectedConversationRead()` ?

Reply:
Yeah. `markAllRead` meant all `messages` in the currently selected `conversation`. When you select a conversation from the LHS we assume you've read all the messages then displayed on the RHS.